service/worker/workerdeployment/workflow_test.go

```diff
@@ -344,10 +348,10 @@ func (s *WorkerDeploymentSuite) syncUnversionedRampInBatches(totalWorkers int) {
 
 	// Mock the SyncDeploymentVersionUserData activity and expect it to be called totalWorkers times
 	var totalBatches int
-	if totalWorkers%syncBatchSize == 0 {
-		totalBatches = totalWorkers / syncBatchSize
+	if totalWorkers%int(s.workerDeploymentClient.getSyncBatchSize()) == 0 {
+		totalBatches = totalWorkers / int(s.workerDeploymentClient.getSyncBatchSize())
 	} else {
-		totalBatches = totalWorkers/syncBatchSize + 1
+		totalBatches = totalWorkers/int(s.workerDeploymentClient.getSyncBatchSize()) + 1
```

Comment:
nit: maybe add `batchSize := s.workerDeploymentClient.getSyncBatchSize()` at the top